Dockerfile

```
@@ -1114,18 +1146,32 @@ COPY --chown=0:0 --from=sdk-rust \
   /usr/share/licenses/rust/
 
 # "sdk-go" has the Go toolchain and standard library builds.
-COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/bin /usr/libexec/go/bin/
-COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/lib /usr/libexec/go/lib/
-COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/pkg /usr/libexec/go/pkg/
-COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/src /usr/libexec/go/src/
-COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/go.env /usr/libexec/go/go.env
-COPY --chown=0:0 --from=sdk-go \
-  /home/builder/sdk-go/licenses/ \
-  /usr/share/licenses/go/
-COPY --chown=0:0 --from=sdk-go \
+ENV GO121VER="1.21.9"
+COPY --chown=0:0 --from=sdk-go-1.21 /home/builder/sdk-go-${GO121VER}/bin /usr/libexec/go-${GO121VER}/bin/
+COPY --chown=0:0 --from=sdk-go-1.21 /home/builder/sdk-go-${GO121VER}/lib /usr/libexec/go-${GO121VER}/lib/
+COPY --chown=0:0 --from=sdk-go-1.21 /home/builder/sdk-go-${GO121VER}/pkg /usr/libexec/go-${GO121VER}/pkg/
+COPY --chown=0:0 --from=sdk-go-1.21 /home/builder/sdk-go-${GO121VER}/src /usr/libexec/go-${GO121VER}/src/
+COPY --chown=0:0 --from=sdk-go-1.21 /home/builder/sdk-go-${GO121VER}/go.env /usr/libexec/go-${GO121VER}/go.env
+COPY --chown=0:0 --from=sdk-go-1.21 \
+  /home/builder/sdk-go-${GO121VER}/licenses/ \
+  /usr/share/licenses/go-${GO121VER}/
+
+COPY --chown=0:0 --from=sdk-go-1.21 \
   /home/builder/aws-lc/LICENSE \
   /usr/share/licenses/aws-lc/LICENSE
 
+
+ENV GO122VER="1.22.1"
```

Can make this more generic so the same copy commands can be reused (copy/pasted) in other stages:
```suggestion
ENV GOVER="1.22.1"
```